server/service/src/sync/translations/stock_line.rs

```diff
@@ -128,6 +128,7 @@ impl SyncTranslation for StockLineTranslation {
             item_variant_id,
             donor_id,
             vvm_status_id,
+            campaign_id: None,
```

Comment:
Can you create a follow up issue for sync, or implement sync here?
We also need to create an issue in the mSupply repo to create the campaign_id field so that it can be synced